src/cli_plugin/install/kibana.js

```diff
@@ -19,34 +17,28 @@ export function existingInstall(settings, logger) {
 
 export async function rebuildCache(settings, logger) {
   logger.log('Optimizing and caching browser bundles...');
-  const serverConfig = _.merge(
-    readYamlConfig(settings.config),
-    {
-      env: 'production',
-      logging: {
-        silent: settings.silent,
-        quiet: !settings.silent,
-        verbose: false
-      },
-      optimize: {
-        useBundleCache: false
-      },
-      server: {
-        autoListen: false
-      },
-      plugins: {
-        initialize: false,
-        scanDirs: [settings.pluginDir, fromRoot('src/core_plugins')]
-      },
-      uiSettings: {
-        enabled: false
-      }
-    }
-  );
-
-  const kbnServer = new KbnServer(serverConfig);
-  await kbnServer.ready();
-  await kbnServer.close();
+
+  const kibanaScript = /^win/.test(process.platform)
```

Comment:
thoughts on if this is going to cause dev vs prod issues?

`node scripts/kibana_plugin install` is going to exec bin/kibana which is going to use the the bin node install?

Reply:
Huh, I did not expect `node scripts/kibana_plugin` to exist... I don’t think this matters enough to block 6.3 but we can discuss it if anyone notices

Reply:
https://github.com/elastic/kibana/blob/6.3/bin/kibana#L18.  confirming..we have the `which node` fallback in 6.3, but in 6.x/7.0 we'll get `unable to find usable node.js executable.`

Reply:
@jbudz addressed this in https://github.com/elastic/kibana/pull/19426/commits/7e752c02e1ca93f095b7bb3179a974f491f7f350